include/psa/crypto_extra.h

```diff
@@ -2069,6 +2069,264 @@ static inline struct psa_pake_operation_s psa_pake_operation_init(void)
     return v;
 }
 
+typedef enum {
+/** An invalid key data format identifier. */
+    PSA_KEY_DATA_FORMAT_NONE,
+/**
+ *  DER or PEM encoded RSAPublicKey data structure as defined in RFC 8017
+ *  (PKCS#1) with:
+ *
+ *  RSAPublicKey ::= SEQUENCE {
+ *       modulus          INTEGER, -- n
+ *       publicExponent   INTEGER  -- e }
+ *
+ *  Key attributes when importing:
```

Comment:
It would help if it was clearer that this is about the way the supplied key attributes are used. Perhaps "When importing a key in this format, the supplied key attributes are used as follows:"